package.json

```diff
@@ -135,7 +135,7 @@
     "redis": "^3.1.1",
     "semver": "^7.5.4",
     "tough-cookie": "^4.1.3",
-    "browserify-sign": "^4.2.2"
+    "browserify-sign": "^4.2.1"
```

Comment:
This was set to 4.2.2 because of a CVE recommendation, did CVE-2023-46234 not reappear with the downgrade?